apps/client/src/routes/dashboards/dashboards-index/components/migration.tsx

```diff
@@ -0,0 +1,54 @@
+import { memo } from 'react';
+import {
+  ExpandableSection,
+  Box,
+  Button,
+  ColumnLayout,
+} from '@cloudscape-design/components';
+import { FormattedMessage } from 'react-intl';
+import { colorBackgroundHomeHeader } from '@cloudscape-design/design-tokens';
+import { dashboardMigration } from '~/services';
+
+const click = () => {
+  async () => {
+    await dashboardMigration();
+  };
+};
+
```

Comment:
Consider using Tanstack Query to interface with the API, which will provide a higher-level interface, configuration options, and API integration utilities. There are some examples in iot-application and many more in the iot-app-kit dashboard query editor modules you can follow.

Reply:
Yes, I plan on adding tanstack query next.